x-pack/plugins/security_solution/public/management/pages/policy/store/services/ingest.ts

```diff
@@ -148,7 +148,7 @@ export const sendGetFleetAgentsWithEndpoint = (
     query: {
       page: 1,
       perPage: 1,
-      kuery: 'fleet-agents.packages : "endpoint"',
+      kuery: 'fleet-agents.packages : "endpoint" and fleet-agents.last_checkin >= "now-89d"',
```

Comment:
Any chance we can get this `89` value from the actual transform definition?
That would scale a little better should we change that in the future or should the transform be changed in a deployment.

Reply:
I don't think that's.. possible? In the transform, that number shows up here: https://github.com/elastic/endpoint-package/blob/master/package/endpoint/elasticsearch/transform/metadata_current/default.json#L7

That json file is not pre-processed, so that `now-90d` line there can't pull from some other file, source, variable, etc. It's just text in that json file.

So then telling _other_ places about it.. remains difficult. Since it's not in some central `DEFINES` file or something we can try to look at from kibana. We can't really parse it out out the package.

The best bet, would be to access that transform definition. Either by seeing if we can go through fleet to get the current installed package files.. and then parse that JSON file, and get to `source.query.@timestamp.range.gt` where we get the string `"now-90d/d"`, or we get the same thing, by accessing the current running transform definition, wherein we also go through the same fields to end up with the string `""now-90d/d"`, which doesn't look fun to parse. Because though it is simple now, and we know what the string is and where the number is, it can change to anything. out of band from kibana releases.